charts/consul/templates/gossip-encryption-autogenerate-job.yaml

```
@@ -13,6 +13,7 @@ metadata:
     chart: {{ template "consul.chart" . }}
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
+    component: gossip-encryption-autogeneneration
```

To match the job name:
```suggestion
    component: gossip-encryption-autogenenerate
```